### ISSUE TITLE
[FEATURE REQUEST]: Multiple NWO/NWK Watershed Additions

### ISSUE THREAD

**Issue by adamscarberry** [USACE/cumulus#373]
email request

> Hello,
> 
> I would like to make another request to add to the NWO office watersheds and create an NWK office and add watersheds. The 2 zips attached should have all the basin extents needed. 
> 
> For NWO, we would be comfortable deleting the 3 currently existing (Above Fort Peck, Big Horn, and Above Canyon Ferry). We would like to add the extents in the provided NWO Watersheds.7z:
> 
>     BFR as Belle Fourche River
>     Bighorn
>     Canyon Ferry
>     FR as Fall River
>     GR as Grand River
>     HR as Heart River
>     JR as James River
>     Lower South Platte
>     Marias
>     North Platte
>     Papio Creek
>     RC as Rapid Creek
>     Salt Creek
>     Upper South Platte
> 
> For NWK we would like to add this as an office choice, and include the following watersheds provided in the NWK Watersheds.zip
> 
>     Chariton
>     Little Platte
>     Lower Kansas
>     Osage
>     Upper Kansas

**Comment by adamscarberry** [comment]
## NWK Watersheds

### Chariton
```
xmin = 167000
xmax = 307000
ymin = 1800000
ymax = 2020000
```

<img width="673" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/c4c6ef91-efa6-45ca-ac55-7da3f30d36eb">


### Little Platte
```
xmin = 69000
xmax = 161000
ymin = 1785000
ymax = 1957000
```

<img width="622" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/e04f885a-8065-426e-89f3-0d36483be677">


### Lower Kansas
```
xmin = -682000
xmax = 235000
ymin = 1689000
ymax = 2050000
```

<img width="427" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/96aaaece-59a0-4e7a-b64e-22f1beaba7f0">


### Osage
```
xmin = -36700
xmax = 412000
ymin = 1538000
ymax = 1839000
```

<img width="487" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/86918cc6-76b1-4474-a58c-5c324e13798c">


### Upper Kansas
```
xmin = -682000
xmax = -47200
ymin = 1689000
ymax = 2038000
```

<img width="572" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/b17d910e-a7e3-4ad1-a6c2-f9febea0fbe3">


In summary, all of NWK below:

<img width="779" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/65dff1c9-464e-4ab8-b174-06644c2df3c8">


Verification of Watersheds names in Cumulus for selection:

![image](https://github.com/USACE/cumulus/assets/46494851/2ba2b6e9-e38e-4aa4-b4cc-125cf6a8d0fd)

**Comment by adamscarberry** [comment]
- [x] Deploy NWK watersheds to develop
- [x] Deploy NWK watersheds to stable

**Comment by adamscarberry** [comment]
## NWO Watersheds

### Belle Fourche River

```
xmin = -810000
xmax = -588000
ymin = 2313000
ymax = 2493000
```

<img width="685" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/ed7f04c7-8283-41b7-8ef2-99148385fda6">


### Bighorn

```
xmin = -1145000
xmax = -851000
ymin = 2212000
ymax = 2644500
```

<img width="615" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/2542707b-b539-481e-ba56-e2983d2a87e9">


### Canyon Ferry

```
xmin = -1407700
xmax = -1079000
ymin = 2446000
ymax = 2905000
```

<img width="539" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/00bffa1a-2fdf-4591-b7b1-3b23ebba93d1">

### Fall River

```
xmin = -637000
xmax = -576000
ymin = 2270000
ymax = 2341000
```

<img width="609" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/d92de81d-25cc-4e41-94dd-c97ff5ae14c8">

### Grand River

```
xmin = -630500
xmax = -332000
ymin = 2482600
ymax = 2626500
```

<img width="515" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/ed7f2d01-fe3c-412c-9e5e-361c363ffbbf">


### Heart River

```
xmin = -580200
xmax = -351500
ymin = 2588500
ymax = 2727200
```

<img width="619" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/411a8a24-7494-40e7-9034-0d8b9a42d96e">


### James River

```
xmin = -329000
xmax = -132000
ymin = 2533500
ymax = 2781400
```

<img width="589" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/79fda931-9712-4216-9693-4a59bb9ed85e">


### Lower South Platte

```
xmin = -849700
xmax = -371500
ymin = 1793600
ymax = 2100000
```

<img width="706" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/ae050b9d-3646-4ef1-8d5e-1e979556272e">


### Marias

```
xmin = -1317100
xmax = -1047100
ymin = 2786700
ymax = 3010400
```

<img width="585" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/89b18271-94f4-44c8-b105-5bc3a3ccd6a7">

### North Platte

```
xmin = -1092900
xmax = -371500
ymin = 1951400
ymax = 2310300
```

<img width="560" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/1ac60ddf-0ead-4bda-99a8-2f3772a9413a">

### Papio Creek

```
xmin = -44000
xmax = 29000
ymin = 1989600
ymax = 2084900
```

<img width="477" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/3a8a3534-f539-440a-8dda-68f26f3c0a50">


### Rapid Creek

```
xmin = -655200
xmax = -529700
ymin = 2324100
ymax = 2405800
```

<img width="488" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/44324d02-ada6-4696-bebb-f39324f5ca49">


### Salt Creek

```
xmin = -105000
xmax = -5800
ymin = 1927900
ymax = 2065600
```

<img width="446" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/52b041cb-4e89-48c0-a006-5094129adc30">


### Upper South Platte

```
xmin = -891100
xmax = -710000
ymin = 1771700
ymax = 1935100
```

<img width="690" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/f47ffcd7-8f89-4a11-a0e8-a5ff18d53d90">

Summary below:

<img width="788" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/b276157c-854c-4a40-8de4-b0fd187357a0">


User Interface Select Options:

<img width="577" alt="image" src="https://github.com/USACE/cumulus/assets/46494851/e5899161-061b-4249-b248-401ec9bf0440">

**Comment by adamscarberry** [comment]
- [x] Deploy to develop
- [x] Deploy to stable

**Comment by adamscarberry** [comment]
deployed
